pkg/mqtrigger/mqtrigger.go

```diff
@@ -40,10 +45,57 @@ func Start(logger *zap.Logger, routerUrl string) error {
 	// Message queue type: nats is the only supported one for now
 	mqType := os.Getenv("MESSAGE_QUEUE_TYPE")
 	mqUrl := os.Getenv("MESSAGE_QUEUE_URL")
+
+	// For authentication with message queue
+	secrets, err := readSecrets(logger)
```

Comment:
We should check whether the `TLS_ENABLED` is true. If not, we should not call the function.

Reply:
I think checking for `TLS_ENABLED` makes sense but I was thinking of `readSecrets` as a generic way to read any kind of secrets for a message queue (these secrets can be unrelated to TLS). What do you think?

Reply:
@life1347 , what do you think of this ^ ?

Reply:
make sense